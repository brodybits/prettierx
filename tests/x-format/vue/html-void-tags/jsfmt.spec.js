// [prettierx merge update from prettier@2.3.1]
run_spec(__dirname, ["vue"], { errors: { vue: true } });
run_spec(__dirname, ["vue"], {
  htmlVoidTags: true,
  errors: { vue: true },
});
