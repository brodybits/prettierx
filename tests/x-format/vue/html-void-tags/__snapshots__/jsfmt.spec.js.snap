// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input.html [vue] format 1`] = `
"Unexpected character \\"EOF\\" (2:1)
  1 | <input type='text'>
> 2 |
    | ^"
`;

exports[`input.html - {"htmlVoidTags":true} [vue] format 1`] = `
"Unexpected character \\"EOF\\" (2:1)
  1 | <input type='text'>
> 2 |
    | ^"
`;
